src/components/editor/markdown-renderer/markdown-it-plugins/task-lists.ts

```diff
@@ -0,0 +1,143 @@
+// Markdown-it plugin to render GitHub-style task lists; see
+//
+// https://github.com/blog/1375-task-lists-in-gfm-issues-pulls-comments
+// https://github.com/blog/1825-task-lists-in-all-markdown-documents
+
+import MarkdownIt from 'markdown-it/lib'
+import StateCore from 'markdown-it/lib/rules_core/state_core'
+import Token from 'markdown-it/lib/token'
+
+let disableCheckboxes = true
+let useLabelWrapper = false
+let useLabelAfter = false
+
+interface TaskListsOptions {
+  enabled: boolean
+  label: boolean
+  labelAfter: boolean
+}
+
+type TokenConstructorType = typeof Token
```

Comment:
Is this type necessary? Iirc you could just use "new Token"

Reply:
That's a good question. The constructor got passed around in the markdown-it-task-lists as well.

Appears to work by just using Token.